data/glsk/glsk-document-api/src/main/java/com/farao_community/farao/data/glsk/api/util/converters/GlskPointScalableConverter.java

```diff
@@ -70,6 +73,27 @@ public static Scalable convert(Network network, AbstractGlskPoint glskPoint) {
         }
     }
 
+    private static void convertRemainingCapacity(Network network, AbstractGlskShiftKey glskShiftKey, List<Float> percentages, List<Scalable> scalables) {
```

Comment:
Why is it only for remaining capacity down for now ? Would it work in all cases ? Maybe add some comments to be more explicit on these points

Reply:
Remaining capacity is, by essence, not symmetrical for up and down scale. Waiting for PowSyBl to provide UpDownScalable (next release), I only created one of the two scalables (the one that was useful for our first process using this data).

Reply:
I will detail the comment below

Reply:
Ok so for this case (B44 = ReserveGSKBlock if I'm not mistaken) you consider only decreasing production and for merit order (B45 = MeritOrderGSKBlock) you consider only increasing production ? And so for now if we do a negative variation on a merit order or a positive variation on a reserve block, results would be wrong ? 

I do now get it that there is no way to define a scalable that would be asymmetrical for up/down variations. But if what I said above is correct maybe you can add a bit of comments to explicit the direction that is handled for now for each type of gsk block. I just fear that if we have to refactor in few weeks to handle all cases properly we get a bit lost if it's not you that implements the features.